inc/Engine/Preload/Database/Tables/Cache.php

```diff
@@ -14,6 +14,7 @@ class Cache extends Table {
 	public function __construct() {
 		parent::__construct();
 		add_action( 'rocket_preload_activation', [ $this, 'maybe_upgrade' ] );
+		add_action( 'wp_rocket_upgrade', [ $this, 'maybe_upgrade' ] );
```

Comment:
why is this one not enough to fix the issue? This should be triggered on every load, front and back, right?

Reply:
how about hooking it on init in there instead of wp_rocket_upgrade? That way it will run on front too

Reply:
it won't be noticeable, the upgrade method bails out early if there is nothing to do

Reply:
Awesome, let's do it like that and see how it goes with testing